pvlib/iotools/solcast.py

```diff
@@ -0,0 +1,448 @@
+""" Functions to access data from the Solcast API.
+"""
+
+import requests
+import pandas as pd
+from dataclasses import dataclass
+
+
+BASE_URL = "https://api.solcast.com.au/data"
+
+
+@dataclass
+class ParameterMap:
+    solcast_name: str
+    pvlib_name: str
+    conversion: callable = lambda x: x
+
+
+# define the conventions between Solcast and PVLib nomenclature and units
+VARIABLE_MAP = [

```

Comment:
A few more renames are needed, I think:
- clearsky_dhi -> dhi_clear
- clearsky_dni -> dni_clear
- clearsky_ghi -> ghi_clear
- clearsky_gti -> poa_global_clear (??)

@AdamRJensen any thoughts on `clearsky_gti`?

Reply:
I suppose this is consistent with the usage of ``poa_global`` and ``_clear``. What are you thinking @kandersolar ?

Reply:
Was mostly wondering if we already had a name for clear-sky GTI, and if not, looking for a second opinion that `poa_global_clear` is a good choice :)